common-graphql/src/main/scala/queries/common/ObsQueriesGQL.scala

```diff
@@ -54,13 +54,7 @@ object ObsQueriesGQL:
                 id
               }
             }
-            target {
-              id
-              sidereal {
-                ra $AngleSubquery
-                dec $AngleSubquery
```

Comment:
Were we even using the ra/dec from here before?

Reply:
No it wasn't used